[pytest]
python_files = *utils.py
; python_functions = test_sum_positive_nums

при добавлении в код маркера @pytest.mark.negative не получается провести тесты без ошибки

также не работает фильтрация в файле .ini не смог разобраться с чем это связано, какой-то сбой в среде разработки